docs/how-to-work-on-tutorials-that-use-coderoad.md

```diff
@@ -0,0 +1,103 @@
+- Make a PR to `main` using your new `feat/version-X.X.Y` branch. Give it a title of `version X.X.Y ready for review`. This will not be merged, it is just to let reviewers know that there is a new version ready
+- Leave it here for reviewers
+
+### Testing changes to a version branch
+- Following the instructions on the [rdb-alpha repo](https://github.com/freeCodeCamp/rdb-alpha) to run a container
+- Start the tutorial using the `tutorial.json` file on whatever fork the changes are on. Making sure to use the file on the `feat: version-X.X.Y` branch and not the `main` branch
+
+### Pushing a new version
+Before pushing a new version, view the new `feat/version-vX.X.Y` (will be merged to `main`) branch on the user's fork and make sure there are additions to the `CHANGELOG.md` file that include the new changes, and the version in the two spots of `coderoad.yaml` matches the new version branch.
+
+If you have write access to the freeCodeCamp repo, have verified the `CHANGELOG` and `coderoad.yaml` files, have tested the changes using the instructions above, and want to push a new version of a tutorial:
+
+> [!WARNING]
+>
+> Reminder. Never make or push changes to a version branch that is on one of the freeCodeCamp repos. Always create a new one
+
+- If you don't have a remote to where the new changes exist, create a remote to the user's fork with `git remote add <users_fork>`
+- Delete any **local** branches that share a name with the new branches. Likely named either `vX.X.Y` or `feat/version-X.X.Y`
+- Checkout the new version branch with `git checkout -b vX.X.Y <remote>/vX.X.Y`
+- Push the new version branch to the freeCodeCamp repo with `git push -u upstream/vX.X.Y`. You need to push the new branch before you update `main` with the new `tutorial.json` file
+- Checkout the users branch that will be merged into `main` with `git checkout -b feat/version-X.X.Y <remote>/feat/version-X.X.Y`
+- Change the `uri` in `coderoad.yaml` back to the freeCodeCamp repo
+- Add and commit the changes
+- Run `coderoad build` to create the new `tutorial.json` file
+- Add and commit the file
+- Push the changes to your fork with `git push -u origin/feat/version-X.X.Y`
+- Make a PR to `main` on the freeCodeCamp repo
+- If you are satisfied, merge it or leave it and ask for a review from someone
+- After the PR is merged, it's probably a good idea to open the tutorial, following the instructions on the [rdb-alpha repo](https://github.com/freeCodeCamp/rdb-alpha), to make sure it's loading properly and you can get through a few steps
+- Finally, if any PR's for this version exists, close them
+
+### How to revert to a previous version
+- Create a new branch off the latest main with `git checkout -b revert/to-version-X.X.X`
+- Revert all commits on this branch until one commit before the version after you want to rever to. For example, you may have commits that look like this:
```

Comment:
I took a crack at clarifying this, though I may have made things more confusing:

```suggestion
- Revert all commits on this branch up to and including the commit of the version after the one you want to revert to. For example, you may have commits that look like this:
```

It's a tough concept to explain, but your example helps clear things up a lot. Feel free to disregard this if you'd like to stick with the original wording.